src/transport/SecureSessionMgr.cpp

```diff
@@ -82,6 +82,21 @@ CHIP_ERROR SecureSessionMgr::Init(NodeId localNodeId, System::Layer * systemLaye
     return err;
 }
 
+Transport::Type SecureSessionMgr::GetTransportType(NodeId peerNodeId)
+{
+    PeerConnectionState * state = mPeerConnections.FindPeerConnectionState(peerNodeId, nullptr);
+
+    if (state)
+    {
+        Transport::PeerAddress peerAddr = state->GetPeerAddress();
```

Comment:
This copies the `PeerAddress`.  Why do we want to do that?  Why not just return either `state->GetPeerAddress().GetTransportType()` or use `const Transport::PeerAddress &` (or `auto &`) for the type of the temporary?

Reply:
Acked